deploy/operator/gather.sh

```diff
@@ -129,15 +129,15 @@ function gather_capi_data() {
     done
   done
 
-  oc logs deployment/operator -n hypershift > "${capi_dir}/hypershift.log" || true
+  oc logs --tail=-1 deployment/operator -n hypershift > "${capi_dir}/hypershift.log" || true
 
   # Detect namespace where the capi-provider lives. The specific name depends whether deployed standalone or via
   # Hypershift, that's why we grep for both.
   capi_provider_ns=$(oc get pods --no-headers -A -o jsonpath='{range .items[*]}{@.metadata.name}{" "}{@.metadata.namespace}{"\n"}' | egrep "capi-provider|cluster-api-provider-agent" | awk -F " " '{print $2}' | head -n1) || true
   capi_provider_dir="${capi_dir}/${capi_provider_ns}"
   mkdir -p "${capi_provider_dir}"
 
-  oc get pods -n ${capi_provider_ns} -o=custom-columns=NAME:.metadata.name --no-headers | xargs -r -I {} sh -c "oc logs {} -n ${capi_provider_ns} --all-containers > ${capi_provider_dir}/logs_{}.log" || true
+  oc get pods -n ${capi_provider_ns} -o=custom-columns=NAME:.metadata.name --no-headers | xargs -r -I {} sh -c "oc logs --tail=-1 {} -n ${capi_provider_ns} --all-containers > ${capi_provider_dir}/logs_{}.log" || true
```

Comment:
same question. I don't see any proof for it to be truncated:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_assisted-service/3989/pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-operator-capi/1539256049951838208/artifacts/e2e-metal-assisted-operator-capi/assisted-baremetal-operator-gather/artifacts/capi/assisted-spoke-cluster-assisted-test-cluster/